src/test/java/uk/gov/hmcts/reform/orgrolemapping/servicebus/TopicConsumerTest.java

```diff
@@ -32,7 +32,7 @@ public void setUp() {
         sut = new TopicConsumer(bulkAssignmentOrchestrator, deserializer);
     }
 
-    @Test
+    //@Test
```

Comment:
This test is wasting 1 minute of time in pipeline and local build without bringing any benefit.

Reply:
Yes, we know it was added for line coverage. we can disable it for now.